org.eclipse.winery.frontends/app/topologymodeler/src/app/edmm/assign-deployment-technology.component.ts

```diff
@@ -0,0 +1,62 @@
+/********************************************************************************
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ ********************************************************************************/
+
+import { Component, Input, OnInit } from '@angular/core';
+import { NgRedux } from '@angular-redux/store';
+import { TNodeTemplate } from '../models/ttopology-template';
+import { IWineryState } from '../redux/store/winery.store';
+import { WineryActions } from '../redux/actions/winery.actions';
+import { BackendService } from '../services/backend.service';
+import { DeploymentTechnology } from '../models/deployment-technology';
+
+@Component({
+    selector: 'winery-assign-deployment-technology',
+    templateUrl: './assign-deployment-technology.component.html',
+    styleUrls: ['./assign-deployment-technology.component.css']
+})
+export class AssignDeploymentTechnologyComponent implements OnInit {
+
+    static QNAME = '{http://www.opentosca.org/winery/extensions/tosca/2013/02/12}deployment-technology';
```

Comment:
Please also use a similar concept for the namespace here. It is, e.g., also used here: `AssignParticipantsComponent`